package.json

```diff
@@ -98,7 +98,7 @@
     "mouse-change": "^1.4.0",
     "mouse-event-offset": "^3.0.2",
     "mouse-wheel": "^1.2.0",
-    "ndarray": "^1.0.18",
+    "ndarray": "^1.0.19",
```

Comment:
This is support for `BigInt64Array` types and 5 days old.
I prefer opening pull requests for the modules listed below:
```
ndarray-pack
ndarray-scratch
simplicial-complex-contour
gl-contour2d
gl-cone3d
gl-line3d
gl-mesh3d
gl-scatter3d
gl-plot2d
gl-plot3d
gl-select-static
gl-buffer
gl-texture2d
vectorize-text
```

Reply:
Reverted the `ndarray` version in https://github.com/plotly/plotly.js/pull/4475/commits/d182131d7bb513a61e3b259434bcaaf8e369f3b6 for now.
We could bump that one when the following PRs are merged; and then we could patch gl-vis modules as well.
https://github.com/mikolalysenko/vectorize-text/pull/16
https://github.com/scijs/ndarray-scratch/pull/7
https://github.com/scijs/ndarray-pack/pull/6
https://github.com/mikolalysenko/simplicial-complex-contour/pull/1

Reply:
Good call. Thanks @archmoj !